spec/unit/puppetfile_spec.rb

```diff
@@ -323,12 +304,11 @@ def expect_wrapped_error(orig, pf_path, wrapped_error)
         block.call
       end
 
-      mod1 = spy('module')
+      mod1 = instance_double('R10K::Module::Base', :cachedir => :none)
+      mod2 = instance_double('R10K::Module::Base', :cachedir => :none)
```

Comment:
hmm okay I understand the `spy`->`double` change now, but why did you now only change these to `instance_double`s? Is there something particular about this test that makes it relevant here but not for the other tests you updated?

Reply:
Nope! Just didn't know about verifying doubles until our conversation. Pretty much all of these doubles could be upgraded to verifying doubles. I learned something new doing some research about spies. 🙂